geetools/ee_profiler.py

```diff
@@ -92,12 +92,20 @@ def _to_dict(self, input: str) -> dict:
             else desc,  # Description can have multiple words
         }
 
+        # The default splitting splits the description and only takes
+        # the first word so we need to join the trailing words dropped by the zip.
+        fix_description = False
+        if headers[-1] == "Description":
+            fix_description = True
```

Comment:
could be done in 1 line: 

```suggestion
        fix_description = headers[-1] == "Description"
```